src/modelbench/benchmark_runner.py

```diff
@@ -485,7 +552,9 @@ def _check_ready_to_run(self):
 
     def run(self) -> BenchmarkRun:
         self._check_ready_to_run()
```

Comment:
Do you want to collect the RuntimeErrors from both annotators and suts here so that all issues can be addressed simultaneously? If I'm understanding correctly, right now, it won't check the annotators if there's an issue with the suts.

Reply:
I was thinking to keep them separate, as right now our annotators can take up to 15 minutes to decide whether or not they're ready. This will get people quick notification of a SUT problem. But if we can get off HuggingFace to something faster, then that would make sense to me.